plugins/qcheck-stm/src/stm_of_ir.ml

```diff
@@ -582,7 +585,61 @@ let cmd_type ir =
     type_declaration ~name:(noloc "cmd") ~params:[] ~cstrs:[]
       ~kind:(Ptype_variant constructors) ~private_:Public ~manifest:None
   in
-  pstr_type Recursive [ { td with ptype_attributes = [ show_attribute ] } ]
+  pstr_type Recursive [ td ]
+
+let pp_cmd_case value =
+  let lhs = mk_cmd_pattern value in
+  let qualify_pp = qualify [ "Util"; "Pp" ] in
+  let get_name id = Option.fold ~none:"()" ~some:str_of_ident id |> evar in
```

Comment:
I’d suggest instead:
```ocaml
  let get_name = function
    | None -> eunit
    | Some id -> evar (str_of_ident id)
  in
```
so that we use the correct `eunit` rather than the ill-formed `evar "()"` (even if it might be pretty-printed the same way)